CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ### Features
 
-- Show "To be defined" if date of the next Catalyst fund is in the past ([PR 3105](https://github.com/input-output-hk/daedalus/pull/3105))
```

Comment:
Replacing CHANGELOG entry on purpose - effectively, what was delivered in this unreleased PR, was deleted